### DISCUSSION TITLE
Proper changelogs

### DISCUSSION THREAD

**Discussion by Andre601** [darkreader/darkreader#8441] [Ideas]
I'm following the releases of this repository to see when a new DarkReader release is published.
The issue however is, that those releases give 0 information on what is new, what was removed and what has changed overall.
It's a bit sad to not have some more clear information regarding this and leave people like me in the dark (which in this case isn't positive).

So, I would like to propose a change to the release system to include information regarding what has changed since the past release. Actions to f.e. list merged Pull requests and commits already exist for this, so there is pretty much no excuse for that to not be implemented.

**Comment by Gusted** [comment]
CC @alexanderby

**Comment by alexanderby** [comment]
Hi, I've added a Change Log, you can see it in project's [root](https://github.com/darkreader/darkreader/blob/master/CHANGELOG.md)
